src/test/java/io/debezium/connector/yugabytedb/YugabyteDBExplicitCheckpointingTest.java

```diff
@@ -143,90 +143,4 @@ public void verifyCommitOffsetCheckpointAndGetCheckpointBehaviour(boolean consis
         // Stop the engine started in this test.
         engine.stop();
     }
-
-    @ParameterizedTest
-    @MethodSource("io.debezium.connector.yugabytedb.TestHelper#streamTypeProviderForStreaming")
```

Comment:
This test was redundant and took ~13 minutes to run so I have removed it to save some time.